python/dgl/nn/pytorch/conv/cugraph_relgraphconv.py

```diff
@@ -309,57 +163,58 @@ def forward(self, g, feat, etypes, norm=None):
             so any input of other integer types will be casted into int32,
             thus introducing some overhead. Pass in int32 tensors directly
             for best performance.
-        norm : torch.Tensor, optional
```

Comment:
Is this replaced by `apply_norm=True`?

Reply:
Yes. This is an additional feature from CuGraphRelGraphConv: supporting the normalized aggregation presented in the RGCN paper.

Hence, users no longer need to compute the norm during training:
https://github.com/dmlc/dgl/blob/465828cdbc1adf570b4aa6c4922cc03bf7abdef2/examples/pytorch/rgcn/entity_sample.py#L60-L61